app/assets/stylesheets/actiontext.scss

```diff
@@ -3,7 +3,7 @@
 // the trix-editor content (whether displayed or under editing). Feel free to incorporate this
 // inclusion directly in any other asset bundle and remove this file.
 //
-@import "trix/dist/trix";
+//= require trix/dist/trix
```

Comment:
why did you change it to require?

Reply:
I got error in rich text area. So to figure it out I changed from import to require. Now changed it properly